tests/test-cases/Group6-VIC-Machine/6-10-List.robot

```diff
@@ -83,14 +83,3 @@ List with valid datacenter
     ${ret}=  Run  bin/vic-machine-linux ls --target %{TEST_URL}/%{TEST_DATACENTER} --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=%{TEST_PASSWORD}
     Set Environment Variable  TEST_DATACENTER  ${orig}
     Verify Listed Machines  ${ret}
-
-List with empty cluster
```

Comment:
@jzt I would suggest creating an issue to add this test to nightly test suite.

Reply:
https://github.com/vmware/vic/issues/7839 👍